sphinx_click/ext.py

```diff
@@ -515,7 +636,7 @@ def run(self) -> ty.Iterable[nodes.section]:
                 command.strip() for command in self.options.get('commands').split(',')
             ]
 
-        return self._generate_nodes(prog_name, command, None, nested, commands)
+        return self._generate_nodes(prog_name, command, None, nested, commands, hide_header=hide_header)
```

Comment:
Don't use the named kwarg here. It's not necessary

Reply:
Yes, I added that because of something I later removed and forgot to change it back.
I'll change it back.

Reply:
Ack